neck_yaw:
  ID: 1
  Return_Delay_Time: 0
  CW_Angle_Limit: 0
  CCW_Angle_Limit: 1023
  Default_Moving_Speed: 50


neck_pitch:
  ID: 2
  Return_Delay_Time: 0
  CW_Angle_Limit: 402
  CCW_Angle_Limit: 602
  Default_Moving_Speed: 50
 

neck_roll:
  ID: 3
  Return_Delay_Time: 0
  CW_Angle_Limit: 420
  CCW_Angle_Limit: 620
  Default_Moving_Speed: 50

base_yaw:
  ID: 4
  Return_Delay_Time: 0
  CW_Angle_Limit: 0
  CCW_Angle_Limit: 1023
  Default_Moving_Speed: 50


base_pitch:
  ID: 5
  Return_Delay_Time: 0
  CW_Angle_Limit: 402
  CCW_Angle_Limit: 602
  Default_Moving_Speed: 50
 

base_roll:
  ID: 6
  Return_Delay_Time: 0
  CW_Angle_Limit: 420
  CCW_Angle_Limit: 620
  Default_Moving_Speed: 50